content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Installing Istio for virtual machine connectivity
+description: Install an Isito mesh such that a virtual machine can connect to it and configure the virtual machine.
+weight: 3
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to install an Istio control plane
+[multicluster deployment](/docs/ops/deployment/deployment-models/#multiple-clusters) and connect your first virtual machine.
+
+## Prerequisites
+
+- One or more Kubernetes clusters with versions: {{< supported_kubernetes_versions >}}.
+
+- Authority to [deploy the Istio control plane](/docs/setup/install/istioctl/).
```

Comment:
nit: I don't think this aligns with our other install docs, and I am not sure many will understand what this means. Seems like we could copy from https://deploy-preview-7248--preliminary-istio.netlify.app/docs/setup/install/istioctl/#prerequisites maybe?

Reply:
suggestion integrated - nice spot!